cinema_service/settings.py

```diff
@@ -116,6 +116,15 @@
 
 AUTH_USER_MODEL = "user.User"
 
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": (
+        "rest_framework.authentication.TokenAuthentication",
+    ),
+    "DEFAULT_PERMISSION_CLASSES": (
+        "rest_framework.permissions.AllowAny",
```

Comment:
You can change it to `IsAdminOrIfAuthenticatedReadOnly`

And remove `authentication_classes `and `permission_classes `definition from views